openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -458,9 +458,14 @@ export const getAddCustomPropertyPath = (entityTypeFQN: string) => {
   return path;
 };
 
-export const getCustomEntityPath = (entityTypeFQN: string) => {
-  let path = ROUTES.CUSTOM_ENTITY_DETAIL;
-  path = path.replace(PLACEHOLDER_ENTITY_TYPE_FQN, entityTypeFQN);
+export const getCustomPropertyPath = (entityTypeFQN: string) => {
+  let path = ROUTES.SETTINGS_WITH_TAB;
+  path = path
+    .replace(
+      PLACEHOLDER_SETTING_CATEGORY,
+      GlobalSettingsMenuCategory.CUSTOM_ATTRIBUTES
+    )
+    .replace(PLACEHOLDER_ROUTE_TAB, entityTypeFQN);
```

Comment:
We might have a method in route.utils to get path for CustomProperty
It will be getGloablSettingsPath something

Reply:
It's the same function as before but the path it was returning was the old one. I just updated it and changed its name.

Reply:
Done